reV/econ/economies_of_scale.py

```diff
@@ -0,0 +1,306 @@
+        """Capital cost found in the data input arg scaled by the evaluated
+        EconomiesOfScale input equation.
+
+        Returns
+        -------
+        out : float | np.ndarray
+            Capital cost found in the data input arg scaled by the evaluated
+            EconomiesOfScale equation.
+        """
+        cc = copy.deepcopy(self.raw_capital_cost)
+        cc *= self.capital_cost_scalar
+        return cc
+
+    @property
+    def fcr(self):
+        """Fixed charge rate from input data arg
+
+        Returns
+        -------
+        out : float | np.ndarray
+            Fixed charge rate from input data arg
+        """
+        key_list = ['fixed_charge_rate', 'mean_fixed_charge_rate',
+                    'fcr', 'mean_fcr']
+        return self._get_prioritized_keys(self._data, key_list)
+
+    @property
+    def foc(self):
+        """Fixed operating cost from input data arg
+
+        Returns
+        -------
+        out : float | np.ndarray
+            Fixed operating cost from input data arg
+        """
+        key_list = ['fixed_operating_cost', 'mean_fixed_operating_cost',
+                    'foc', 'mean_foc']
+        return self._get_prioritized_keys(self._data, key_list)
+
+    @property
+    def voc(self):
+        """Variable operating cost from input data arg
+
+        Returns
+        -------
+        out : float | np.ndarray
+            Variable operating cost from input data arg
+        """
+        key_list = ['variable_operating_cost', 'mean_variable_operating_cost',
+                    'voc', 'mean_voc']
+        return self._get_prioritized_keys(self._data, key_list)
+
+    @property
+    def aep(self):
+        """Annual energy production from input data arg. If a valid key is not
+        found in the input data, aep is calculated as:
+
+        AEP = (System Capacity (MW) * mean capacity factor
+               * 8,760 hours/year * 1,000 kWh/MWh)
+
+        Returns
+        -------
+        out : float | np.ndarray
+            Annual energy production from input data arg
+        """
+
+        key_list = ['aep', 'annual_energy_production', 'mean_aep',
+                    'mean_annual_energy_production']
+        if any([k in self._data for k in key_list]):
+            aep = self._get_prioritized_keys(self._data, key_list)
+        else:
+            cap = self._get_prioritized_keys(self._data, ['capacity'])
+            cf = self._get_prioritized_keys(
+                self._data, ['mean_cf', 'capacity_factor'])
+            aep = cap * cf * 8760 * 1000
+
+        return aep
+
+    @property
+    def raw_lcoe(self):
+        """LCOE calculated with the unscaled (raw) capital cost
+
+        LCOE = (FCR * raw_capital_cost + FOC) / AEP + VOC
+
+        Returns
+        -------
+        lcoe : float | np.ndarray
+            LCOE calculated with the unscaled (raw) capital cost
+        """
+        lcoe = ((self.fcr * self.raw_capital_cost + self.foc) / self.aep
```

Comment:
Do we want to make LCOE a static method with the required args and call it here and in the below property?

Reply:
Yeah that's not a bad idea in case someone just wants the LCOE equation. Where do you think is a good place for the static method? I almost think its most appropriate in the Econ class even though it would only get used here.

Reply:
ugg, thats a good question. My first thought was in the EconomyOfScale class, but now that you mention it what about putting it in utilities? If we ever envision it being a stand-alone function, might as well make it one.